cunumeric/config.py

```diff
@@ -16,49 +16,218 @@
+    CUNUMERIC_MAX_TASKS: int
+    CUNUMERIC_NONZERO: int
+    CUNUMERIC_POTRF: int
+    CUNUMERIC_RAND: int
+    CUNUMERIC_READ: int
+    CUNUMERIC_RED_ALL: int
+    CUNUMERIC_RED_ANY: int
+    CUNUMERIC_RED_ARGMAX: int
+    CUNUMERIC_RED_ARGMIN: int
+    CUNUMERIC_RED_CONTAINS: int
+    CUNUMERIC_RED_COUNT_NONZERO: int
+    CUNUMERIC_RED_MAX: int
+    CUNUMERIC_RED_MIN: int
+    CUNUMERIC_RED_PROD: int
+    CUNUMERIC_RED_SUM: int
+    CUNUMERIC_REPEAT: int
+    CUNUMERIC_SCALAR_UNARY_RED: int
+    CUNUMERIC_SORT: int
+    CUNUMERIC_SYRK: int
+    CUNUMERIC_TILE: int
+    CUNUMERIC_TRANSPOSE_COPY_2D: int
+    CUNUMERIC_TRILU: int
+    CUNUMERIC_TRSM: int
+    CUNUMERIC_TUNABLE_HAS_NUMAMEM: int
+    CUNUMERIC_TUNABLE_MAX_EAGER_VOLUME: int
+    CUNUMERIC_TUNABLE_NUM_GPUS: int
+    CUNUMERIC_TUNABLE_NUM_PROCS: int
+    CUNUMERIC_TYPE_POINT1: int
+    CUNUMERIC_TYPE_POINT2: int
+    CUNUMERIC_TYPE_POINT3: int
+    CUNUMERIC_TYPE_POINT4: int
+    CUNUMERIC_TYPE_POINT5: int
+    CUNUMERIC_TYPE_POINT6: int
+    CUNUMERIC_TYPE_POINT7: int
+    CUNUMERIC_TYPE_POINT8: int
+    CUNUMERIC_TYPE_POINT9: int
+    CUNUMERIC_UNARY_OP: int
+    CUNUMERIC_UNARY_RED: int
+    CUNUMERIC_UNIQUE_REDUCE: int
+    CUNUMERIC_UNIQUE: int
+    CUNUMERIC_UNLOAD_CUDALIBS: int
+    CUNUMERIC_UOP_ABSOLUTE: int
+    CUNUMERIC_UOP_ARCCOS: int
+    CUNUMERIC_UOP_ARCCOSH: int
+    CUNUMERIC_UOP_ARCSIN: int
+    CUNUMERIC_UOP_ARCSINH: int
+    CUNUMERIC_UOP_ARCTAN: int
+    CUNUMERIC_UOP_ARCTANH: int
+    CUNUMERIC_UOP_CBRT: int
+    CUNUMERIC_UOP_CEIL: int
+    CUNUMERIC_UOP_CLIP: int
+    CUNUMERIC_UOP_CONJ: int
+    CUNUMERIC_UOP_COPY: int
+    CUNUMERIC_UOP_COS: int
+    CUNUMERIC_UOP_COSH: int
+    CUNUMERIC_UOP_DEG2RAD: int
+    CUNUMERIC_UOP_EXP: int
+    CUNUMERIC_UOP_EXP2: int
+    CUNUMERIC_UOP_EXPM1: int
+    CUNUMERIC_UOP_FLOOR: int
+    CUNUMERIC_UOP_FREXP: int
+    CUNUMERIC_UOP_GETARG: int
+    CUNUMERIC_UOP_IMAG: int
+    CUNUMERIC_UOP_INVERT: int
+    CUNUMERIC_UOP_ISFINITE: int
+    CUNUMERIC_UOP_ISINF: int
+    CUNUMERIC_UOP_ISNAN: int
+    CUNUMERIC_UOP_LOG: int
+    CUNUMERIC_UOP_LOG10: int
+    CUNUMERIC_UOP_LOG1P: int
+    CUNUMERIC_UOP_LOG2: int
+    CUNUMERIC_UOP_LOGICAL_NOT: int
+    CUNUMERIC_UOP_MODF: int
+    CUNUMERIC_UOP_NEGATIVE: int
+    CUNUMERIC_UOP_POSITIVE: int
+    CUNUMERIC_UOP_RAD2DEG: int
+    CUNUMERIC_UOP_REAL: int
+    CUNUMERIC_UOP_RECIPROCAL: int
+    CUNUMERIC_UOP_RINT: int
+    CUNUMERIC_UOP_SIGN: int
+    CUNUMERIC_UOP_SIGNBIT: int
+    CUNUMERIC_UOP_SIN: int
+    CUNUMERIC_UOP_SINH: int
+    CUNUMERIC_UOP_SQRT: int
+    CUNUMERIC_UOP_SQUARE: int
+    CUNUMERIC_UOP_TAN: int
+    CUNUMERIC_UOP_TANH: int
+    CUNUMERIC_UOP_TRUNC: int
+    CUNUMERIC_WHERE: int
+    CUNUMERIC_WINDOW_BARLETT: int
+    CUNUMERIC_WINDOW_BLACKMAN: int
+    CUNUMERIC_WINDOW_HAMMING: int
+    CUNUMERIC_WINDOW_HANNING: int
+    CUNUMERIC_WINDOW_KAISER: int
+    CUNUMERIC_WINDOW: int
+    CUNUMERIC_WRITE: int
+    CUNUMERIC_ZIP: int
```

Comment:
Could move this to a `types.py` or similar module. I don't think it can go in a `typings` folder